draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.tsx

```diff
@@ -6,15 +6,15 @@ import { IconButton } from "@kaizen/button"
 import { Heading } from "@kaizen/typography"
 import chevronUp from "@kaizen/component-library/icons/chevron-up.icon.svg"
 import chevronDown from "@kaizen/component-library/icons/chevron-down.icon.svg"
-import { Sticky } from "./CollapsibleGroup"
+import { Sticky } from "../CollapsibleGroup/CollapsibleGroup"
 import styles from "./Collapsible.module.scss"
 
 type Variant = "default" | "clear"
 
 export interface CollapsibleProps
   extends OverrideClassName<HTMLAttributes<HTMLDivElement>> {
   id: string
-  children: JSX.Element | JSX.Element[] | string
+  children: React.ReactNode
```

Comment:
hmmm I've somewhat flip flopped on this, pretty sure its not necessary and I'm hesitant to change a type that could cause issues for consumers the line since ReactNode can be a few more things 🤔 Maybe lets just keep that as is for now

Reply:
Totally understandable. I'll change it back to `JSX.Element | JSX.Element[] | string`